### DISCUSSION TITLE
Adopt eslint-plugin-astro ☔

### DISCUSSION THREAD

**Discussion by barziahmed** [withastro/roadmap#858] [Proposal]
Hello, 

Astro has a strong advantage where it holds its intergreations and plugins for tools (i.e. Prettier) and frameworks as its own and first-party projects, which makes them reliable and trustworthy, but there is one tool that I want which is community-driven, I'm of course not its bad or anything, and I absolutey have all respect for its author and maintainers, but I suggest that Astro adopt it into its organization and puts it under its umbrella and develop it officially, so it joins its sisters like withastro/prettier-plugin-astro.

**Comment by Princesseuh** [comment]
In all honesty, ota-meshi is a genius of the ESLint ecosystem. None of us in the team have worked on ESLint plugins, and we wouldn't be able to really give anything to it. I already maintain the Prettier plugin alone and it's generally tough to balance with working on Astro itself and the language-tools repo (which I also kinda maintain alone)

We'd much rather the credit go entirely to ota-meshi at this time, and if he's one day less active we'll take it over only then.
